storage/container_storage_interface/osd-persistent-storage-aws-efs-csi.adoc

```diff
@@ -9,31 +9,31 @@ toc::[]
 
 [IMPORTANT]
 ====
-This procedure is specific to the Amazon Web Services Elastic File System (AWS EFS) CSI Driver Operator, which is only applicable for {product-title} 4.10 and later versions.
+This procedure is specific to the Amazon Web Services Elastic File System (Amazon EFS) CSI Driver Operator, which is only applicable for {product-title} 4.10 and later versions.
 ====
 
 == Overview
 
-{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for AWS Elastic File Service (EFS).
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Amazon Elastic File Service (EFS).
 
 Familiarity with link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/storage/index#persistent-storage-overview_understanding-persistent-storage[persistent storage] and link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/storage/index#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
 
-After installing the AWS EFS CSI Driver Operator, {product-title} installs the AWS EFS CSI Operator and the AWS EFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the AWS EFS CSI Driver Operator to create CSI-provisioned PVs that mount to AWS EFS assets.
+After installing the Amazon EFS CSI Driver Operator, {product-title} installs the Amazon EFS CSI Operator and the Amazon EFS CSI driver by default in the `openshift-cluster-csi-drivers` namespace. This allows the Amazon EFS CSI Driver Operator to create CSI-provisioned PVs that mount to Amazon EFS assets.
```

Comment:
Revoked.

Reply:
@smunje1 It doesn't seem like this title is linking out to OCP resources or code that reference "AWS EFS". "Amazon EFS CSI driver" is the appropriate name to be used for the driver. Is there a reason I'm missing why this update should be revoked?

https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html

Reply:
Revoking for Line 21 seems correct here.